### DISCUSSION TITLE
How to change preset fan and gas?

### DISCUSSION THREAD

**Discussion by fatrabbit-la** [artisan-roaster-scope/artisan#1028] [General]
I'm using Artisan on a Mill City 1kg. The fan and gas default to values of 50 and 20. They used to default to values of 41 and 10 which is my preference. Any idea how to change this?

1) I have no idea how they started defaulting to the previous preferable values in the first place. If that's something I set, I have no idea how I did it.

2) Given the above, I further have no idea how they changed from the previous to the current defaults.

I'm pretty consistent in how I save files at the end of roasts but are they somehow retained as defaults when saving files a certain way?
![22_1202_1032_ETH_Gersi](https://user-images.githubusercontent.com/94999664/205419529-ccf56cb5-8ee1-4ab8-a800-53afe10e040e.png)

I'm attaching an example. In this case I had to go into Events and modify to Fan 41 and Gas 0.5 (10 in the Events window) but these numbers just show up and I'm glad they do. I don't manually enter them.

Dan

**Comment by poundy** [comment]
if you're doing back to back roasting, and just going from OFF to a START, what you leave the roaster set at carries over to your start of roast. I also as part of my BBP tend to set gas and airflow a fair way from where they ended the roast (fan from 60 down to 35, burner from 1 to 31 and back to 5 for charge) so I know well before I charge that my measurements are recorded appropriately
